src/jit/lower.cpp

```diff
@@ -2809,7 +2860,7 @@ GenTree* Lowering::LowerNonvirtPinvokeCall(GenTreeCall* call)
 
     InsertPInvokeCallProlog(call);
 
-    if (call->gtCallType != CT_INDIRECT)
+    if ((comp->opts.eeFlags & CORJIT_FLG_PINVOKE_DIRECT_CALLS) == 0 && call->gtCallType != CT_INDIRECT)
```

Comment:
Why checking for CORJIT_FLG_PINVOKE_DIRECT_CALLS flag?
Btw, in my source search i couldn't find this flag anywhere.  What am i missing?